libbeat/outputs/console/console.go

```diff
@@ -47,15 +47,15 @@ func (c *console) Close() error {
 func (c *console) PublishEvent(
 	s op.Signaler,
 	opts outputs.Options,
-	event common.MapStr,
+	event outputs.Data,
```

Comment:
can we renamed the field itself also to data? Otherwise we have inception below :-)